src/util/newconfig/ArrayView.hpp

```diff
@@ -112,13 +112,12 @@ class ArrayView {
          * @brief Equality operator
          *
          * @param other Another ArrayIterator to compare
-         * @return true if iterators are equal, otherwise false
+         * @return true if iterators are pointing to the same ArrayView underneath, otherwise false
```

Comment:
True if iterators are pointing to the same element in the same ArrayView object.